### DISCUSSION TITLE
Connector information

### DISCUSSION THREAD

**Discussion by eroyee** [mnh-jansson/open-battery-information#86] [Makita]
Martin, 

Well done on your work. I would like to contribute some information from my batteries (if that's useful) but need to know what the 7-pin connector on the battery is so I can order one? 

Apologies in advance if you've said this somewhere but it wasn't immediately clear to me, and the detail wasn't on your graphic. Thought I'd enquire here as I don't imagine I'll be the only one wanting to know :)

Thank you.

**Comment by Marvv89** [comment]
Most likely that will be [AliExpress](https://www.aliexpress.us/item/3256804743402583.html?src=google&pdp_npi=4%40dis%21EUR%212.52%212.52%21%21%21%21%21%40%2112000031056585092%21ppc%21%21%21&src=google&albch=shopping&acnt=708-803-3821&isdl=y&slnk=&plac=&mtctp=&albbt=Google_7_shopping&aff_platform=google&aff_short_key=UneMJZVf&gclsrc=aw.ds&albagn=888888&ds_e_adid=&ds_e_matchtype=&ds_e_device=c&ds_e_network=x&ds_e_product_group_id=&ds_e_product_id=nl1005004929717335&ds_e_product_merchant_id=106617809&ds_e_product_country=NL&ds_e_product_language=nl&ds_e_product_channel=online&ds_e_product_store_id=&ds_url_v=2&albcp=19207327905&albag=&isSmbAutoCall=false&needSmbHouyi=false&gad_source=1&gclid=Cj0KCQjw5ea1BhC6ARIsAEOG5pwDx4y_Gu7uorOEt93wdzo0-xMKSKa3jslJttTWw6VRr5yhnL--CnQaAq0fEALw_wcB&gatewayAdapt=nld2usa4itemAdapt)

Or the charger side, to connect more easy to a battery:
[Charger connector](https://www.aliexpress.us/item/3256803102549583.html?src=google&pdp_npi=4%40dis%21EUR%214.12%212.08%21%21%21%21%21%40%2112000025038872091%21ppc%21%21%21&src=google&albch=shopping&acnt=272-267-0231&slnk=&plac=&mtctp=&albbt=Google_7_shopping&gclsrc=aw.ds&albagn=888888&ds_e_adid=&ds_e_matchtype=&ds_e_device=c&ds_e_network=x&ds_e_product_group_id=&ds_e_product_id=nl1005003288864335&ds_e_product_merchant_id=493449023&ds_e_product_country=NL&ds_e_product_language=nl&ds_e_product_channel=online&ds_e_product_store_id=&ds_url_v=2&albcp=20701318733&albag=&isSmbAutoCall=false&needSmbHouyi=false&gad_source=1&gclid=Cj0KCQjw5ea1BhC6ARIsAEOG5pw9vWnfy8T-ex-GzoGUf-IjXWi8hEp2R7qjovV3e7GagfEZCXNVUH4aAux0EALw_wcB&aff_fcid=f643b5401a614111ac8fdd4d3d713887-1723458987800-06708-UneMJZVf&aff_fsk=UneMJZVf&aff_platform=aaf&sk=UneMJZVf&aff_trace_key=f643b5401a614111ac8fdd4d3d713887-1723458987800-06708-UneMJZVf&terminal_id=b24f808d58564bcea3110005a94028a1&afSmartRedirect=n&gatewayAdapt=nld2usa4itemAdapt)

**Comment by mnh-jansson** [comment]
@eroyee Thanks! The more information we have the better! If you have pictures of the insides that is also helpful. Currently we do not have a way to identify BMS versions so having the pictures of the BMS and the data, maybe we can see some pattern. ROM ID and battery message are the most useful for this.

The latter of the ones that @Marvv89 linked is the one i use. If you have the battery open you can just poke some wire in the back of the connector if you don't wanna wait for the connector to arrive. Good luck and let us know how it goes!

**Comment by eroyee** [comment]
Thanks to both of you. 

I'd looked on Ali, but wondered if this was a 'standard' connector mildly adapted by Makita, but it appears not (sigh). Will order a couple for the cause, however it may take a few weeks to this location.

In the meantime, for anyone coming here, the pins appear to be standard 0.1" (2.54mm) spacing, and I seem to recall a similar board edge connector in something I've worked on in the past. If I recall what that is I'll report back. 

Otherwise will see what I can do re BMS ID pictures, and I have some other information I'll post separately to this as it doesn't relate to connectors.

**Comment by aldras** [comment]
This reply is a little late, but I would like to add a recommendation, which is what I actually did.
Order the cheapest knock off (made in China?) charger you can find (off Amazon or similar) and then dismantle it to recover the connector.
This will not only give you that special yellow connector, but a stable trouble free locking interface to connect with the battery.
